frontend/test/__support__/e2e/helpers/e2e-collection-helpers.js

```diff
@@ -1,11 +1,5 @@
 import { popover } from "__support__/e2e/cypress";
 
-export function assertCanAddItemsToCollection() {
-  cy.findByTestId("collection-menu").within(() => {
-    cy.icon("add");
-  });
-}
-
```

Comment:
We shouldn't store helpers globally and export them if they are used in one file only.